### PR TITLE
Add Python Getting Started page

### PR DESCRIPTION
**Pull request by zombiezen**
Fixes ch-4767

### CODE REVIEW

**Review by amyhua — APPROVED** [review]
lgtm

File: docs/python-getting-started/index.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+3. Configure your AWS credentials - you can do this one of two ways:
+    1. The standard AWS way (config file, or environment variable, etc), or
+    2. Using a YourBase-specific pair of environment variables (in case you want
+       to have a separate set of credentials just for the graph access), which
+       are `YOURBASE_AWS_ACCESS_KEY_ID` and `YOURBASE_AWS_SECRET_ACCESS_KEY`
+
+## Support for Parallelized Tests
+
+If you've already conducted your CI to parallelize tests, YourBase has you
+covered. YourBase can be configured to work with tests run in cohorts.
```
Comment (glacials) [inline docs/python-getting-started/index.md:184]: Guess we never filled this in!

```suggestion
## Support for Parallelized Tests

If you've already conducted your CI to parallelize tests, YourBase has you
covered. YourBase can be configured to work with tests run in cohorts.

1. Set `YOURBASE_COHORT_COUNT` to the number of shards (or processes, or swarms, etc.) that run your tests. For example:
    ```sh
    export YOURBASE_COHORT_COUNT=12
    ```
2. Set `YOURBASE_ACTIVE_COHORT` to the ID of the current shard, in the range `[1, $YOURBASE_COHORT_COUNT]`. For example:
    ```
    # First shard
    export YOURBASE_ACTIVE_COHORT=1
    # ...
    # Last shard
    export YOURBASE_ACTIVE_COHORT=12
    ```

File: docs/python-getting-started/index.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+
+## Product Usage Data
+
+By default, YourBase tracks how many tests are run and how many are skipped with
+each build. YourBase also tracks the length of the tests. You can opt out of
+data sharing by setting `YOURBASE_TELEMETRY_DISABLED=1`.
```
Comment (glacials) [inline docs/python-getting-started/index.md:190]: This is what's released actually:
```suggestion
data sharing by setting `YOURBASE_TELEMETRY=false`.
```

**Review by glacials — APPROVED** [review]

**Comment by zombiezen** [comment]
Thanks @glacials! I checked in 2e4e22ee64a84183f819d14abd1322ac81747554 to address those suggestions.

**Pull request merged**